align.go

```diff
@@ -170,7 +166,7 @@ func (a *Align) columnLength() []string {
 	return lines
 }
 
-// export will pad each field in lines based on the Aligner's column counts
+// export will pad each field in lines based on the Align's column counts
```

Comment:
```suggestion
// export will pad each field in lines based on the Align's column counts.
```

Reply:
Fixed - in a few other comments as well.